dev-python/matrix2latex/matrix2latex-1.0.0.ebuild

```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+PYTHON_COMPAT=( python{2_7,3_3,3_4} )
+
+inherit distutils-r1 multilib
+
+DESCRIPTION="A tool to create LaTeX tables from python lists and arrays"
+HOMEPAGE="https://github.com/TheChymera/matrix2latex"
+SRC_URI="https://github.com/TheChymera/${PN}/blob/master/archive/${PN}Python-${PV}.tar.gz?raw=true -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+RDEPEND=""
+DEPEND=""
```

Comment:
*DEPEND should be dropped. Although empty IUSE could in principle also be dropped, I like to have it set empty.